packages/api/lambdas/ems-report.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const moment = require('moment');
+const { generateReports } = require('../lib/ems');
+
+/**
+ * handler, generate daily report
+ *
+ * @param {Object} event - event passed to lambda
+ * @param {Object} context - AWS Lambda context
+ * @param {*} callback - callback function
+ * @returns {undefined} - none
+ */
+function handler(event, context, callback) {
+  // 24-hour period ending past midnight
+  const endTime = moment.utc().startOf('day').toDate().toUTCString();
```

Comment:
Instead of converting to a string, could you pass the timestamp as a moment object?  You're creating a moment object, converting it to a string, passing it into the function, then converting it back into a moment object before it's used.